tests/runtests.py

```diff
@@ -233,6 +234,22 @@ def actual_test_processes(parallel):
         return parallel
 
 
+def validate_selenium(selenium_args):
+    """
+    Validate the browsers requested and raise appropriate exception.
+    """
+    # Remove any browser repition.
```

Comment:
If the alphabetically first browser is duplicated, we would have the tests running for it twice. It is not a big thing though. That's the only problem duplicate browsers will bring up.